src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+.search-bar {
+  margin: 20px 0;
+}
+
+.search-bar__input {
+  padding-left: 62px;
+  display: block;
+  height: 70px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 26px 25px;
+  outline: none;
+}
+
+.search-bar__input::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar__input--small {
+  height: 42px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input:focus {
+  background: url(./images/Search.svg) 26px 25px no-repeat,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.search-bar__input--small:focus {
+  background: url(./images/Search.svg) 13px 15px no-repeat,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
you repeat here `no-repeat,
    linear-gradient(180deg, #fff 0%, #f6f6f7 100%)`
you may skip it under second selector